src/classnames/tailwind.ts

```diff
@@ -7386,7 +7393,10 @@ export type TWidth =
   | 'w-card'
   | 'w-mobile-card'
   | 'w-full-125'
+  | 'w-full-105'
+  | 'w-screen-93'
   | 'w-screen-80'
+  | 'w-screen-45'
```

Comment:
Should we stick to the screen sizes?

Reply:
It's an easy way to handle resizing without messing with `min` and `max` width

Reply:
MinMax is hard to adapt on all screen sizes + have resizing that won't be triggered by inner changes (for example, expanding Email block when you have `min-max` + `width-full` will expand the block itself, but this shouldn't happen, this is why we should strictly limit the `width`, but keep it resizeable on different screens, which can be achieved only by using `vw`